source/views/menus/components/fancy-menu.js

```diff
@@ -200,6 +199,7 @@ export class FancyMenu extends React.Component<Props, State> {
 				renderSectionHeader={this.renderSectionHeader}
 				sections={(groupedMenuData: any)}
 				style={styles.inner}
+				windowSize={5}
```

Comment:
Do we need to change this prop on Menu? Or is this just, like, to be careful? I'd like to only override it when we actually feel like we need to…

Reply:
I did it because it prevents a noticeable lag when you turn off the "Specials Only" filter. It seemed to have a similar issue to the Course Search ListView where it was rendering a ton of rows right away...

Reply:
Kk

Reply:
Kk